src/SprykerEco/Client/Payone/PayoneClient.php

```diff
@@ -176,8 +175,7 @@ public function initPaypalExpressCheckout(PayoneInitPaypalExpressCheckoutRequest
     }
 
     /**
-     * Send genericpayment request to payone with action "getexpresscheckoutdetails"
```

Comment:
Copy this description to the interface.

Reply:
The one in ClientInterface is already more general, so I've decided to skip it.